docs/Developer-Guide.md

```diff
@@ -212,11 +212,14 @@ $ sudo systemctl restart systemd-journald
 >
 > - You should only do this step if you are testing with the latest version of the agent.
 
-The rust-agent is built with a static linked `musl.` To configure this:
+The agent is built with a statically linked `musl.` The default `libc` used is `musl`, but on `ppc64le` and `s390x`, `gnu` should be used. To configure this:
 
 ```
-rustup target add x86_64-unknown-linux-musl
-sudo ln -s /usr/bin/g++ /bin/musl-g++
+$ export ARCH=$(uname -m)
+$ if [ "$ARCH" = "ppc64le" -o "$ARCH" = "s390x" ]; then export LIBC=gnu; else export LIBC=musl; fi
+$ [ ${ARCH} == "ppc64le" ] && export ARCH=powerpc64le
+$ rustup target add ${ARCH}-unknown-linux-${LIBC}
+$ [ ${LIBC} == "musl" ] && sudo ln -s /usr/bin/g++ /bin/musl-g++
```

Comment:
While we're at it (sorry :sweat_smile:), this `ln` appears to be a relic from the Go agent. Rust does not require this.

Reply:
@Jakob-Naucke  @snir911 could you help me confirm this `ln -s /usr/bin/g++ /bin/musl-g++` not requre?
https://github.com/kata-containers/kata-containers/blob/6a9e5f90f7b69c3c519087106f3b136e3253852b/tools/osbuilder/scripts/lib.sh#L275

Reply:
#3682